sdk/src/platform/Platform.ts

```diff
@@ -424,6 +428,10 @@ export default class Platform extends EventEmitter {
     }
 
     public async inflateRequest(request: Request, options: SendOptions = {}): Promise<Request> {
+        if (this._handleRateLimit) {
+            options.handleRateLimit = this._handleRateLimit;
+        }
+
```

Comment:
It should be below `options = options || {}` otherwise you risk to assign handleRateLimit to undefined...